src/main/java/org/opensearch/timeseries/util/ParseUtils.java

```diff
@@ -363,29 +361,6 @@ public static SearchSourceBuilder generatePreviewQuery(
         return new SearchSourceBuilder().query(detector.getFilterQuery()).size(0).aggregation(dateRangeBuilder);
     }
 
-    public static String generateInternalFeatureQueryTemplate(AnomalyDetector detector, NamedXContentRegistry xContentRegistry)
```

Comment:
I removed this method since it is not used: https://github.com/search?q=repo%3Aopensearch-project%2Fanomaly-detection%20generateInternalFeatureQueryTemplate&type=code